### PR TITLE
Big Xeno Update

### PR DESCRIPTION
**Pull request by Guidesu**
Xenos are Buffed and nerfed in several ways.
🆑They gain a few xenos ability.
They gain they own set of organs.
They are space proof and a little cold proof. Cold still applies slowdown on they.
They take SEVERE burns damage and heat damage and have brute modifier to 2x because they can regenerate on weeds.
They can persons through walls but can be flashed even with sec or sunglasses googles
The organs heals VERY in a very slow way.

### CODE REVIEW

**Comment by faaaay** [comment]
> Makes they Rad-proof.
> Makes they inorganic because they are

Are xenos rad-proof or inorganic in the lore? This doesn't ring a bell. It's a good PR, but I do kind of feel like they'd still be pretty strong without those two.

**Comment by MitztheKat** [comment]
I'll consider merging this once you make the suggestions that gawb suggests. Xenos shouldn't be too OP.

**Comment by Guidesu** [comment]
Radiation proof i understand but they are inogarnic.
They are not like normal sentient life.
They don't catch any type of disease or anything because they have acid blood and an totally different biology
How an virus could attack cells if they are acid

**Comment by Guidesu** [comment]
Done

**Comment by faaaay** [comment]
Looks pretty good. Strong, but not massively so, and it has drawbacks.

**Comment by faaaay** [comment]
Actually - I do see some grammatical errors in the descriptions. Mind if I leave suggestions on the commit?

**Comment by Guidesu** [comment]
Go ahead

File: code/modules/surgery/organs/ears.dm [hunk 1]
```diff
@@ -118,6 +118,14 @@
 	damage_multiplier = 0.1 //STRONK
 	bang_protect = 1 //Fear me weaklings.
 
+/obj/item/organ/ears/xx
+	name = "purple ears"
+	desc = "an strange set of ears"
```
Comment (MitztheKat) [inline code/modules/surgery/organs/ears.dm]: Add a .

File: code/modules/surgery/organs/eyes.dm [hunk 2]
```diff
@@ -154,6 +154,16 @@
 	see_in_dark = 8
 	sight_flags = SEE_MOBS | SEE_OBJS | SEE_TURFS
 
+/obj/item/organ/eyes/xx
+	name = "Xenomorphic Eyes"
+	desc = "Ew its having an pulsating goo."
```
Comment (MitztheKat) [inline code/modules/surgery/organs/eyes.dm]: "Eww, It has pulsating goo."

File: code/modules/surgery/organs/heart.dm [hunk 3]
```diff
@@ -96,10 +96,11 @@
 		failed = TRUE
 
 obj/item/organ/heart/slime
-	name = "slime heart"
-	desc = "It seems we've gotten to the slimy core of the matter."
+	name = "Squish heart"
```
Comment (MitztheKat) [inline code/modules/surgery/organs/heart.dm]: squish heart? really? we have an xeno heart right?

File: code/modules/surgery/organs/heart.dm [hunk 4]
```diff
@@ -226,6 +227,14 @@ obj/item/organ/heart/cybernetic/upgraded/on_life()
 	desc = "An electronic pump that regulates hydraulic functions, the electronics have EMP shielding."
 	icon_state = "heart-c"
 
+
+/obj/item/organ/heart/xx
+	name = "Xeno heart"
```
Comment (MitztheKat) [inline code/modules/surgery/organs/heart.dm]: Not capitalized correctly, both the name and desc.

File: code/modules/surgery/organs/lungs.dm [hunk 5]
```diff
@@ -477,6 +477,18 @@
 	safe_toxins_max = 0
 	maxHealth = INFINITY//I don't understand how plamamen work, so I'm not going to try t give them special lungs atm
 
+/obj/item/organ/lungs/xx
+	name = "air filter"
+	desc = "A sponge mass for filtering gases in the air."
```
Comment (MitztheKat) [inline code/modules/surgery/organs/lungs.dm:482]: "A spongey mass for filtering gases in the air."

File: code/modules/surgery/organs/stomach.dm [hunk 6]
```diff
@@ -92,11 +92,17 @@
 	icon_state = "stomach-x" //xenomorph liver? It's just a black liver so it fits.
 	desc = "A mutant stomach designed to handle the unique diet of a flyperson."
 
+/obj/item/organ/stomach/xx
+	name = "Xeno stomach"
+	icon_state = "stomach-x" //xenomorph liver? It's just a black liver so it fits.
+	desc = "A mutant stomach designed to handle the unique diet of a xenomorph"
```
Comment (MitztheKat) [inline code/modules/surgery/organs/stomach.dm]: forgot a period.

File: code/modules/surgery/organs/ears.dm (hunk 1 above)
Comment (Guidesu) [reply code/modules/surgery/organs/ears.dm]: done

**Review by MitztheKat — CHANGES_REQUESTED** [review]
Maybe this'll show up now

File: code/modules/surgery/organs/heart.dm (hunk 3 above)
Comment (Guidesu) [reply code/modules/surgery/organs/heart.dm]: yup we have just lazy

**Comment by MitztheKat** [comment]
close enough. I can patch the rest in a future update. We can push this now.

**Pull request merged**